doc/workshops/scientific-computing.rst

```diff
@@ -2,4 +2,800 @@
+   num_observations = len(dog_or_cat_pivot)
+   dfn = num_groups - 1
+   dfd = num_observations - num_groups
+
+   f_critical = stats.f.ppf(q=0.95, dfn=dfn, dfd=dfd)
+
+The value of ``f_critical`` (2.6241) is larger than that of ``f_value`` (0.5813). This means that the variance between the means of these groups is not significantly different. Since the *p* value indicates that we cannot rule out random variations, then we have to discard our hypothesis.
+
+What is the conclusion of all this? There seems to be little or no evidence in our data to think that there is indeed a correlation between a person's pet preferences and his or her ability to solve practical or abstract problems. Cat and dog owners, rejoice!
+
+
+
+================
+Report and share
+================
+
+In science, sharing results, good and bad, is critical. Today, in scientific computing, it is also crucial to share data, and the code used to process and analyze it. To help with this, we will use Spyder's Code Analysis.
+
+
+Code Analysis
+~~~~~~~~~~~~~
+
+When you share code, you want it to be readable, clean, and not overly complex. The Code Analysis component can help us detect these issues and even bugs that can affect the performance of our code.
+
+To see an example, you can open the :download:`scientific-computing-astro.py <scientific-computing-astro.py>` file and run the :guilabel:`Code Analysis` (open the file in the pane and click on the green triangular button in the upper right corner).
+
+.. image:: images/workshop-2/code-analysis-pane-workshop-2.png
+   :alt: Code Analysis pane Example for astrological sign and cognitive ability
+
+You can see four categories that can help you improve your code:
+
+* Convention: programming standard violations
+* Refactor: refactoring related checks
+* Warning: Python-specific problems
+* Error: probable bugs in the code
+
+Each of these categories indicates the type of alert and the line on which the potential issue occurs. For example, it tells us that line 129 is too long, or that we imported numpy but did not use it at all in the code.
+
+These suggestions can contribute to sharing clean code. We recommend using this panel to polish your code before publishing it.
+   
+
+
+===========
+Final words
+===========
+
+In this workshop we have taken the first steps to use Spyder for scientific computing. We have seen how from a question or doubt a hypothesis is raised. This question can also emerge from the exploration of some data. This exploration can be done by calculating some measures (such as mean, median, standard deviation) or by drawing some plots.
+
+Often, in order to answer the question, we must process some data (for example, for the construction of a cognitive ability test).
+
+Finally, we try to establish a relationship between variables with some statistical tests. The results of these tests will support or not our initial hypothesis.
+
+In this workshop you have learned how to:
```

Comment:
I would add a title on top of this. Something like "What we learned".
```suggestion
In this workshop we have learned how to:
```
Also use we instead of you since all the workshop has this kind of language.